lab2/Syntax.in

```diff
@@ -1,18 +1,18 @@
-program ::= {function_declaration} statement 

```

Comment:
Besides converting `::=` into `=` because you are using EBENF instead of BNF, the code is clean an easy to understand.